tests/js/spec/components/timeSince.spec.tsx

```diff
@@ -3,7 +3,8 @@ import {render, screen} from 'sentry-test/reactTestingLibrary';
 import TimeSince from 'sentry/components/timeSince';
 
 describe('TimeSince', function () {
-  const tenMinAgo = new Date(new Date().getTime() - 60000 * 10);
+  const tenMinAgo = new Date(new Date().getTime() - 10 * 60 * 1000);
+
   it('renders a relative date', () => {
     const {rerender} = render(<TimeSince date={new Date()} />);
```

Comment:
Technically if you make new date objects later in tests you can introduce flake, in this case it likely doesn't matter, just mentioning this as a point about dates in ui tests as a pattern since it's simple enough to avoid.

Ideally you start with one static point in time and make all relative dates for testing based off of that.

```js
const now = new Date();
const tenMinAgo = new Date(now.getTime() - ...);
...
......render(<TimeSince date={now} />


```